xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.cpp

```diff
@@ -541,9 +541,9 @@ void CActiveAE::StateMachine(int signal, Protocol *port, Message *msg)
         case CActiveAEControlProtocol::STREAMFFMPEGINFO:
           MsgStreamFFmpegInfo *info;
           info = (MsgStreamFFmpegInfo*)msg->data;
-          par->stream->m_profile = info->profile;
-          par->stream->m_matrixEncoding = info->matrix_encoding;
-          par->stream->m_audioServiceType = info->audio_service_type;
+          info->stream->m_profile = info->profile;
+          info->stream->m_matrixEncoding = info->matrix_encoding;
+          info->stream->m_audioServiceType = info->audio_service_type;
```

Comment:
this does not make sense either. info is the input message. I would assume the data should be copied to some destination. @AlwinEsch what is the goal here?